config/yugabyte/regression_pipelines/orderby_workloads/postgres/ORDG7_orderby_on_hash_partitioned_table.yaml

```diff
@@ -41,14 +41,22 @@ microbenchmark:
                 count: 1
                 rows: 1000000
                 columns:
+                    - name: col_bigint_card1_
+                      count: 1
+                      util: OneNumberFromArray
```

Comment:
Like we have discussed in the past, in such scenarios, we should not use OneNumberFromArray. We should use RandomInt and provide a range. OneNumberFromArray should be used when we dont want discreet numbers and not a range.

Reply:
Done. For cardinality usecase, I wanted 1000 values for each item mentioned in the array. For both the utility function, OneNumberFromArray and RandomInt, the value is never 1000 but in the range of +-100. So replacing OneNumberFromArray with RandomInt seems to ok. Done.